docs/main/docs/about_t3rn/token-economics/tokenomics-inflation.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 1
+---
+
+# Tokenomics and Inflation
+
+**Total token supply**: `100,000,000`
+
+**Token ticker**: `$TRN`
+
+**Inflation**: `4.40%` (yearly)
+
+
+t3rn network participants are able to get `remunerated` through Inflation and Gas fees:
+
+## Inflation 
+
+| Month | Parachain | Collators | Attesters | Executors | Inflation |
+|---|---|---|---|---|---|
+|0|0.2|0.5|0.8|1.1|4.40 %|
+|12|0.2|0.5|0.8|1.1|4.40 %|
+|24|0.2|0.5|0.8|1.1|4.40 %|
+|48|0.2|0.5|0.8|1.1|4.40 %|
+|72|0.2|0.5|0.8|1.1|4.40 %|
+
+## Gas Fees
+
+| | Collators | Executors | Attesters | Authors | Treasury | Users |
+|---|---|---|---|---|---|---|
+| Any TX| 80% | N/A | N/A | N/A | 20% | 0% |
+| Contracts Call TX | 10% | N/A | N/A | 80% | 10% | 0% |
+
+
```

Comment:
single new line ending to all docs

Reply:
[1131805](https://github.com/t3rn/t3rn/pull/519/commits/1131805fb04d61c4bd875e7bdb7929a8080f8cb5)